drivers/service_scanning/app/models/filters.rb

```diff
@@ -0,0 +1,10 @@
+###
```

Comment:
I think this should be removed.

Reply:
The 3 # at the top of the file is in every file.

Reply:
Oh, you meant the file.  I've been having trouble with dev class paths, so these are included to help rails along.